### PR TITLE
Syndicate station 13

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Replaces all job loadouts with syndicate loadouts.
Replaces central command name with Syndicate high command.
Replace weapon charging stations with 1-3 10mm magazines.
Replaces disablers with sketchkin pistols.
Replaces syndicate hardsuits in the uplink with ERT hardsuits.
Nukies are now centcom nerds.
Replaces common radio channel with syndicate channel.
Replaces syndicate space suit in uplink with normal nanotrasen space suit.
Replaces all mapped suit storage units with syndicate varients.

Includes a few new sprites.
image

Includes some new weapons and items too:
 - The recharging belt: A recharger strapped to your belt. Useful for Centcom Operatives who need to charge their low capacity pulse pistols on the go!

## Why It's Good For The Game

Nanotrasen is a boring company, lets embrace the syndicate.

## Changelog
:cl:
tweak: The Syndicate have invaded Nanotrasen space and assumed command of space station 13. All hail the syndicate. Nanotrasen infiltrators are expected on board, watch your backs agents.
fix: Uplink will no longer spawn in freqlocked radios
/:cl:

### CODE REVIEW

**Comment by super12pl** [comment]
Amazing

**Comment by ToasterBan** [comment]
poggers

**Comment by Effemu** [comment]
good pr

**Comment by Mat05usz** [comment]
Finally

**Comment by OceanFish1** [comment]
Gaming

**Comment by ItsTisme** [comment]
epic

**Comment by PowerfulBacon** [comment]
Nukies are now fully replaced with Nanotrasen personelle, with new custom energy based weapons and items to suit their needs.

**Comment by ItsTisme** [comment]
might wana add rubber/sleepy bullets for the sketchin if thats sec main way of fighting crime

**Comment by PowerfulBacon** [comment]
in the syndicate tiders get executed on the spot, but ok as long as they still do some damage

**Comment by NickEmbased** [comment]
spdmrg

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 1]
```diff
@@ -34,3 +34,9 @@
 	name = "10mm incendiary bullet"
 	damage = 15
 	fire_stacks = 2
+
+/obj/item/projectile/bullet/c10mm/rubber
+	name = "10mm bullet"
+	damage = 15
+	stamina = 40
```
Comment (PowerfulBacon) [inline code/modules/projectiles/projectile/bullets/pistol.dm]:
```suggestion
	damage = 10
	stamina = 25
```

File: code/modules/z_syndicate_station/nanotrasen_uplink.dm [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+	syndicate_station_mode = SYNDIE_MODE_ONLY
+
+/datum/uplink_item/dangerous/tesla_revolver
+	name = "Tesla Revolver"
+	desc = "A powerful revolver that uses an internal nuclear fission reactor to generate a massive potential difference \
+		before discharging a ball of energy in whatever direction the gun is facing. Recharges automatically."
+	item = /obj/item/gun/energy/tesla_revolver/self_recharge
+	cost = 6
+	syndicate_station_mode = SYNDIE_MODE_ONLY
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
```
Comment (PowerfulBacon) [inline code/modules/z_syndicate_station/nanotrasen_uplink.dm]:
```suggestion
```

File: _maps/shuttles/infiltrator_basic.dmm [hunk 3]
```diff
@@ -450,7 +450,7 @@
 	movement_force = list("KNOCKDOWN" = 0, "THROW" = 0);
 	name = "syndicate infiltrator";
 	port_direction = 4;
-	width = 23
+	width = 21
```
Comment (PowerfulBacon) [inline _maps/shuttles/infiltrator_basic.dmm]:
```suggestion
	width = 23
```

**Comment by wraith-54321** [comment]
YES but yeah, rubber bullets are a good idea

File: code/game/gamemodes/game_mode.dm [hunk 4]
```diff
@@ -173,9 +173,13 @@
 	else // goals only become purchasable when on_report is called, this also makes a replacement announcement.
 		for(var/datum/station_goal/G in station_goals)
 			G.prepare_report()
+	if(CONFIG_GET(flag/syndicate_station))
+		addtimer(CALLBACK(src, .proc/send_invasion_report, 0), rand(waittime_l, waittime_h))
 	gamemode_ready = TRUE
 	return 1
 
+/datum/game_mode/proc/send_invasion_report()
+	priority_announce("The station has been occupied by the Syndicate and is now under Syndicate command.", "Transfer of Station Ownership", 'sound/ai/intercept.ogg')
```
Comment (PowerfulBacon) [inline code/game/gamemodes/game_mode.dm]:
```suggestion
	priority_announce("The station has been occupied by the Syndicate and is now under Syndicate command. All Nanotrasen members of security and command have been removed from their positions and replaced by Syndicate representatives. All crew members are to follow orders from the syndicate from this point onwards. Glory to the Syndicate.", "Transfer of Station Ownership", 'sound/ai/intercept.ogg')
```

File: code/modules/z_syndicate_station/nanotrasen_uplink.dm [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+/datum/uplink_item/ammo/shotgun_laser
+	name = "Laser Shotgun Rounds"
+	desc = "A small box containing 7 laser shotgun shells."
+	item = /obj/item/storage/box/lasershot
+	cost = 4
+	syndicate_station_mode = SYNDIE_MODE_ONLY
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
```
Comment (PowerfulBacon) [inline code/modules/z_syndicate_station/nanotrasen_uplink.dm]:
```suggestion
/datum/uplink_item/ammo/shotgun
	name = "Lethal Shotgun Rounds"
	desc = "A small box containing 7 lethal shotgun shells."
	item = /obj/item/storage/box/lethalshot
	cost = 1
	syndicate_station_mode = SYNDIE_MODE_ONLY
	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)

/datum/uplink_item/ammo/shotgun_pulse
	name = "Pulse Shotgun Rounds"
	desc = "A small box containing 7 heavy pulse shotgun shells."
	item = /obj/item/storage/box/pulseshot
	cost = 2
	syndicate_station_mode = SYNDIE_MODE_ONLY
	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)

/datum/uplink_item/ammo/shotgun_laser
	name = "Laser Shotgun Rounds"
	desc = "A small box containing 7 laser shotgun shells."
	item = /obj/item/storage/box/lasershot
	cost = 2
	syndicate_station_mode = SYNDIE_MODE_ONLY
	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
```

File: code/modules/z_syndicate_station/nanotrasen_uplink.dm [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+	name = "Energy Gun"
+	desc = "A standard issue energy gun given out to common Nanotrasen security forces."
+	item = /obj/item/gun/energy/e_gun
+	cost = 6
+	syndicate_station_mode = SYNDIE_MODE_ONLY
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+
+/datum/uplink_item/dangerous/shotgun
+	name = "Combat Shotgun"
+	desc = "A self-loading combat shotgun used by Nanotrasen pointmen."
```
Comment (francinum) [inline code/modules/z_syndicate_station/nanotrasen_uplink.dm:84]: It's not automatic anymore.

File: code/modules/z_syndicate_station/nanotrasen_uplink.dm [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+	name = "Tesla Revolver"
+	desc = "A powerful revolver that uses an internal nuclear fission reactor to generate a massive potential difference \
+		before discharging a ball of energy in whatever direction the gun is facing. Recharges automatically."
+	item = /obj/item/gun/energy/tesla_revolver/self_recharge
+	cost = 6
+	syndicate_station_mode = SYNDIE_MODE_ONLY
+
+/datum/uplink_item/dangerous/laser_carbine
+	name = "Laser Carbine"
+	desc = "A high-energy, fully automatic, magasine loaded carbine which fires energy based rounds."
```
Comment (francinum) [inline code/modules/z_syndicate_station/nanotrasen_uplink.dm]:
```suggestion
	desc = "A high-energy, fully automatic, magazine loaded carbine which fires energy based projectiles."
```

File: code/modules/z_syndicate_station/nanotrasen_uplink.dm [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+/datum/uplink_item/dangerous/maeteba
+	name = "Mateba Revolver"
+	desc = "A powerful revolver used by Nanotrasen officers."
+	item = /obj/item/gun/ballistic/revolver/mateba
+	cost = 8
+	syndicate_station_mode = SYNDIE_MODE_ONLY
+
+/datum/uplink_item/ammo/energyclip
+	name = "Energy Clip"
+	desc = "A 20 round rechargable battery compatible with the Nanotrasen laser carbine."
```
Comment (francinum) [inline code/modules/z_syndicate_station/nanotrasen_uplink.dm]:
```suggestion
	desc = "A 20 round rechargeable battery compatible with the Nanotrasen laser carbine."
```

File: code/modules/z_syndicate_station/nanotrasen_uplink.dm [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+	item = /obj/item/ammo_box/magazine/recharge/laser
+	cost = 2
+	syndicate_station_mode = SYNDIE_MODE_ONLY
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+
+/datum/uplink_item/ammo/energyclip_pulse
+	name = "Pulse Energy Clip"
+	desc = "A 16 round rechargable particle pulser which fires destructive pulse lasers. Compatible with the laser carbine"
```
Comment (francinum) [inline code/modules/z_syndicate_station/nanotrasen_uplink.dm]:
```suggestion
	desc = "A 16 round rechargeable pumped laser diode which fires destructive pulse lasers. Compatible with the laser carbine"
```

File: code/modules/z_syndicate_station/nanotrasen_uplink.dm [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+	item = /obj/item/ammo_box/magazine/recharge/laser/pulse
+	cost = 4
+	syndicate_station_mode = SYNDIE_MODE_ONLY
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+
+/datum/uplink_item/ammo/energyclip_highcap
+	name = "High Capacity Energy Clip"
+	desc = "A 38 round rechargable battery compatible with the laser carbine."
```
Comment (francinum) [inline code/modules/z_syndicate_station/nanotrasen_uplink.dm]:
```suggestion
	desc = "A 38 round rechargeable battery compatible with the laser carbine."
```

File: code/modules/z_syndicate_station/nanotrasen_weapons.dm [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+//=======================
+
+/obj/item/ammo_box/magazine/recharge/laser
+	icon_state = "oldrifle"
+	multiple_sprites = 2
+	ammo_type = /obj/item/ammo_casing/energy/laser/heavy
+
+/obj/item/ammo_box/magazine/recharge/laser/high_cap
+	name = "high capacity power pack"
+	desc = "An upgraded, rechargable battery that serves as a magasine for laser carbines."
```
Comment (francinum) [inline code/modules/z_syndicate_station/nanotrasen_weapons.dm]:
```suggestion
	desc = "An upgraded, rechargeable battery that serves as a magazine for laser carbines."
```

File: code/modules/z_syndicate_station/nanotrasen_weapons.dm [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+	icon_state = "riflehighcap"
+	max_ammo = 38
+
+/obj/item/ammo_box/magazine/recharge/laser/pulse
+	name = "pulse power pack"
+	desc = "A power pack for laser carbines, modified "
```
Comment (francinum) [inline code/modules/z_syndicate_station/nanotrasen_weapons.dm]:
```suggestion
	desc = "A power pack for laser carbines, modified to fire high energy pulsed projectiles."
```

**Review by francinum — DISMISSED** [review]

File: code/modules/z_syndicate_station/nanotrasen_uplink.dm (hunk 6 above)
Comment (PowerfulBacon) [reply code/modules/z_syndicate_station/nanotrasen_uplink.dm:84]: It is

**Pull request closed without merging**